### PR TITLE
Updating copyright headers to match current best practices

### LINKED ISSUE #38: Add license header to top of all code files
See https://opensource.google/docs/releasing/licenses/

We can either use;
```
Copyright (C) 2020  The SymbiFlow Authors.

Use of this source code is governed by a ISC-style
license that can be found in the LICENSE file or at
https://opensource.org/licenses/ISC
```

Or put the full license at the top;
```
Copyright (C) 2020  The SymbiFlow Authors.

Permission to use, copy, modify, and/or distribute this software for any
purpose with or without fee is hereby granted, provided that the above
copyright notice and this permission notice appear in all copies.

THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
```

### PR DESCRIPTION
**Pull request by kgugala**
fixes #38

### CODE REVIEW

File: sdf_timing/__main__.py [hunk 1]
```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/env python3
```
Comment (mithro) [inline sdf_timing/__main__.py:1]:
```
#
```

File: sdf_timing/__main__.py [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+# Copyright (C) 2020  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+#
```
Comment (mithro) [inline sdf_timing/__main__.py:9]: Remove extra `#`

Comment (kgugala) [reply sdf_timing/__main__.py]: fixed

**Comment by kgugala** [comment]
- [x] `COPYING` file renamed to `LICENSE` file
- [x] The `LICENSE` file should start with - `Copyright 2020 The SymbiFlow Authors.`
- [x] Add or update the `AUTHORS` files to have the text as shown at https://opensource.google/docs/releasing/authors/
- [x] Make sure all code files have the license specifier at the top of the file as shown at https://opensource.google/docs/releasing/licenses/
- [x] All code files should have `SPDX` identifiers.
- [ ] Add CI which checks for all this.
- [x] Python files should have a `#!/usr/bin/env python3` as the shebang line.
- [x] Python files should have a `# coding: utf-8` as the second line after the shebang.

File: sdf_timing/__main__.py (hunk 1 above)
Comment (kgugala) [reply sdf_timing/__main__.py:1]: fixed

**Comment by kgugala** [comment]
@mithro is there anything in this PR than need to be adressed?

**Pull request merged**